src/lib/storage/zero_suppression/simd_bp128/simd_bp128_decoder.hpp

```diff
@@ -0,0 +1,132 @@
+#pragma once
+
+#include <array>
+#include <memory>
+#include <numeric>
+#include <utility>
+
+#include "storage/zero_suppression/base_zero_suppression_decoder.hpp"
+
+#include "oversized_types.hpp"
+#include "simd_bp128_packing.hpp"
+
+#include "types.hpp"
+
+namespace opossum {
+
+class SimdBp128Vector;
+
+/**
+ * @brief Implements point-access into a SIMD-BP128 compressed vector
+ *
+ * The decoder caches the last decoded block and meta-block.
+ * Performs best if the values are accessed in a cache-coherent manner, i.e.,
```

Comment:
I don't think you mean "cache-coherent"

Reply:
Mhm. I took it from here: http://supercomputingblog.com/optimization/taking-advantage-of-cache-coherence-in-your-programs/ How would you describe it?

Reply:
"In computer architecture, cache coherence is the uniformity of shared resource data that ends up stored in multiple local caches." (Wiki)

What happens here (and in the linked article) is cache-(friendly|aware|sensitive) programming. The use of "cache coherent" in the article is plainly wrong.

Reply:
okay thanks!